pkg/operator/encryption/shared_encryption.go

```diff
@@ -196,15 +206,16 @@ var modeToNewKeyFunc = map[mode]func() []byte{
 	identity:  newIdentityKey,
 }
 
-func getCurrentMode() (mode, error) {
+func getCurrentModeAndExternalReason() (mode, string, error) {
 	// TODO replace this env var hack with a check against some cluster config
+	reason := os.Getenv("ENCRYPTION_EXTERNAL_REASON")
```

Comment:
a backdoor :)?

Reply:
Will allow me to force rollouts in CI and possibly expose this via an API later.